### DISCUSSION TITLE
Security Onion Installation

### DISCUSSION THREAD

**Discussion by jhernandez-sun** [Security-Onion-Solutions/securityonion#6751] [Unsupported Versions] [answered]
Hello!

I need a bit of help. I've been tasked with Security Onion. Reading the documentation, I see that it's recommended to use CentOS 7 or Ubuntu 18.04 LTS. My manager told me that she prefers a RH type environment, so CentOS would be the best choice. Reading on CentOS 7, I found out that the end-of-support is pretty close, so that got me thinking getting CentOS 8 instead of CentOS 7. Unfortunately, I just found out CentOS 8 is dead.

So I continue to read about the other RH-based distros, and got me thinking which one would be a good fit for Security Onion.

Anyways, after babbling too much, and putting aside any CentOS flavor, my many questions are:

- can I use Rocky Linux or Oracle Linux for Security Onion ?
- if I were to install Security Onion on Rocky Linux or Oracle Linux, could I get support?
- should I ditch any RH-based Linux for Ubuntu ?
- Is Ubuntu 18.04 the best option or can I use a newer LTS version ?

I'm looking forward your responses / comments, and any information is highly appreciated.

Thank you.

**Comment by dougburks** [comment, marked as answer]
You can install using our Security Onion ISO image or perform a network installation on CentOS 7, Ubuntu 18.04, or Ubuntu 20.04.

From https://docs.securityonion.net/en/2.3/download.html:
> For most use cases, we recommend using our Security Onion ISO image as it’s the quickest and easiest method.

Our ISO image is based on CentOS 7. CentOS 7 does not reach End Of Life until 2024:
https://wiki.centos.org/About/Product
